doc/source/conf.py

```diff
@@ -52,12 +53,12 @@
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
     "sphinx.ext.napoleon",
-    "sphinx.ext.autodoc.typehints",  # still used with autoapi
+    "sphinx_autodoc_typehints",
```

Comment:
I know this is a reversion, but what's the difference between this and the `sphinx.ext` version?

Reply:
I completely missed this difference. It seems they are in fact two separate extensions: https://github.com/tox-dev/sphinx-autodoc-typehints/issues/167.

Reply:
Comparision of the outputs from both here:
https://github.com/tox-dev/sphinx-autodoc-typehints/issues/167#issuecomment-815968866

Reply:
Ah nice, yes the 3rd party is superior, good to keep it.